### ISSUE TITLE
Throws error in node v8.0.0

### ISSUE THREAD

**Issue by mymattcarroll** [blinkmobile/server-cli#224]
Throws error while running:

```
bm server deploy
```

Stack trace:

```
TypeError: Cannot read property 'pipesCount' of undefined
    at module.exports.Readable.pipe (_stream_readable.js:545:16)
    at module.exports.ZipArchiveOutputStream._smartStream (./path/to/package/server-cli/node_modules/compress-commons/lib/archivers/zip/zip-archive-output-stream.js:184:11)
    at module.exports.ZipArchiveOutputStream._appendStream (./path/to/package/server-cli/node_modules/compress-commons/lib/archivers/zip/zip-archive-output-stream.js:96:20)
    at module.exports.ArchiveOutputStream.entry (./path/to/package/server-cli/node_modules/compress-commons/lib/archivers/archive-output-stream.js:88:10)
    at module.exports.ZipStream.entry (./path/to/package/server-cli/node_modules/zip-stream/index.js:138:49)
    at Zip.append (./path/to/package/server-cli/node_modules/archiver/lib/plugins/zip.js:53:15)
    at Archiver._moduleAppend (./path/to/package/server-cli/node_modules/archiver/lib/core.js:172:16)
    at Archiver._onQueueTask (./path/to/package/server-cli/node_modules/archiver/lib/core.js:370:8)
    at ./path/to/package/server-cli/node_modules/async/dist/async.js:4045:9
    at process (./path/to/package/server-cli/node_modules/async/dist/async.js:2316:17)
```

**Comment by mymattcarroll** [comment]
Issues comes from here this the [archiver](https://www.npmjs.com/package/archiver) npm package. The issue has been reported https://github.com/archiverjs/node-archiver/issues/236

Could potentially be fixed by this pull request https://github.com/nodejs/node/pull/13374 for Node 8.

**Comment by mymattcarroll** [comment]
Looks like this will be fixed in Node 8.1 when this pull request is merged and released:

https://github.com/nodejs/node/pull/13483
